src/writer/logs.py

```
@@ -1,104 +1,249 @@
+    yield buffer
+
+    routing_map.remove_buffer(key)
+
+
+class JSONFormatter(logging.Formatter):
+    def format(self, record: logging.LogRecord, **kwargs) -> str:
+        from writer.blueprints import get_current_block
+        from writer.core import get_app_process, get_session
+
+
+        data: Dict[str, Any] = {
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)